content/pages/mission.md

```diff
@@ -0,0 +1,35 @@
+---
+title: A11yTalks Mission and Values
+template: page
+---
+## Mission
+
+From the website About page: To help bridge the gap between developers and accessibility professionals.
```

Comment:
Remove lines 7-8. "Normalizing digital accessibility through education." is the mission statement.